terraform/main.tf

```diff
@@ -23,41 +23,80 @@ resource "google_compute_subnetwork" "honeypot_subnet" {
   network       = google_compute_network.main.self_link
 }
 
-resource "google_compute_firewall" "allow_ssh_in" {
-  name    = "allow-ssh-in"
+# standard firewall settings
+resource "google_compute_firewall" "firewall_standard_rule" {
+  name    = "firewall-standard-rule"
   network = google_compute_network.main.self_link
 
   allow {
     protocol = "icmp"
   }
-
   allow {
     protocol = "tcp"
-    ports    = ["22", "9091"]
+    ports    = ["22"]
   }
+  source_ranges = ["0.0.0.0/0"]
+}
 
+# open port 9090 and 9091 on our logger-vm: to control metrics and minio
+resource "google_compute_firewall" "firewall_logger_view" {
+  name    = "firewall-logger-view"
+  network = google_compute_network.main.self_link
+  allow {
+    protocol = "tcp"
+    ports    = ["9090", "9091"]
+  }
+  target_tags = [ "observer" ]
```

Comment:
Open port 9090 and 9091 for Observer, so we can have access on MinIO and  Prometheus